packages/aws-cdk-lib/core/lib/cfn-mapping.ts

```diff
@@ -63,30 +63,41 @@ export class CfnMapping extends CfnRefElement {
   }
 
   /**
-   * @returns A reference to a value in the map based on the two keys.
+   * @returns A reference to a value in the map based on the two keys. If mapping is lazy, the value from the map or default value is returned instead of the reference.
    */
-  public findInMap(key1: string, key2: string): string {
+  public findInMap(key1: string, key2: string, defaultValue?: string): string {
```

Comment:
Ohmigod. Can this method be simplified if we reorganize it?

Reply:
Would be glad to, but not seeing a clear way. Think we ship it and someone enterprising can tidy it up later.